src/components/Block/TokenPicker.vue

```diff
@@ -0,0 +1,54 @@
+<script setup>
+import { computed } from 'vue';
+import { formatUnits } from '@ethersproject/units';
+import { _n } from '@/helpers/utils';
+
+const props = defineProps({
+  searchValue: String,
+  assets: Array
+});
+
+const emit = defineEmits(['pick']);
+
+const filteredAssets = computed(() =>
```

Comment:
We could use `fuse.js` to make it smarter (and sort results), but I kept is basic to avoid extra deps.

Reply:
Interesting I didn't knew about Fuse.js